phost/src/chain_client.rs

```diff
@@ -136,6 +136,13 @@ pub fn check_round_end_event(
     Ok(false)
 }
 
+pub async fn get_parachain_heads(client: &XtClient, paraclient: &XtClient, hash: Option<Hash>) -> Option<Vec<u8>> {
+	let para_key = storage_value_key_vec("ParachainInfo", "ParachainId");
```

Comment:
// TODO: cache the ParachainId to save one request on parachain side

Reply:
done

Reply:
Please add //TODO to the code

Reply:
Oh, sorry. It's already cached. Closing this comment